app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -262,5 +266,23 @@ def run_generic_operation(operation, ems_ref, parameters = {})
         end
       end
     end
+
+    def run_specific_operation(operation_name, ems_ref, parameters = {})
```

Comment:
This is a little bit different, the method `run_generic_operation` is used for invoking the operation 
`ExecuteOperation` which receives a DMR operation name as a part of its parameters and executes it. But that only removes the Datasource from EAP7 and not from the Inventory.

I need to invoke a specific operation, `RemoveDatasource`, in order to delete the Datasource from EAP and Inventory, for this propose, I've created this method and uses `invoke_specific_operation` inside it.